### ISSUE TITLE
assignment variable missing 

### ISSUE THREAD

**Issue by priya-anand** [openemr/openemr#529]
File Path - openemr-master/library/edihistory
File Name - test_edih_835_accounting.php

Line.no. 407 - `.= (isset($sar[3]) && $sar[3]) ? trim($sar[3]) : '0';`

**Comment by stephenwaite** [comment]
quite a few little typos you've found, i wonder if @kevmccor brought in the wrong couple files since both names start with test?

**Comment by kevmccor** [comment]
I got an e-mail.  I guess since you used my ID.  The "test_" files are unfinished and unused.  I was trying to duplicate the accounting from the /billing/era_payments.php code in this one.  The test_sftp file is for using sftp to automate file downloads.  I don't have any way to really test it but I worked out how to process the downloaded files for csv data and move them into appropriate directories.  Just rough drafts in there in case someone wants to pick it up. @stephenwaite @priya-anand
